bluesky_queueserver/manager/plan_monitoring.py

```diff
@@ -120,8 +120,8 @@ def start(self, doc):
             uid = doc["uid"]
             self._run_list.add_run(uid=uid)
 
-            print(f"New run was open: '{uid}'")
-            print(f"Run list: {self._run_list.get_run_list()}")
+            logger.info(f"New run was open: '{uid}'")
+            logger.debug(f"Run list: {self._run_list.get_run_list()}")
```

Comment:
I saw you used the `%s` style for logging above, which is considered more performant. Do you want to do the same here?

Reply:
Yes